judgments/utils/search_utils.py

```diff
@@ -16,7 +16,7 @@ def _valid_years():
     Generate a list of valid years as strings.
     """
     today = datetime.date.today()
-    valid_years = range(2003, today.year)
+    valid_years = range(2003, today.year + 1)
```

Comment:
Oh `range`, why don't you do what anyone expects? 😄

Reply:
jokingly: we should add a linter rule that `range(x)` is invalid, it should either be `range(x+1)` or `range(x+0)` if you really mean it.